tir/technologies/poui_internal.py

```diff
@@ -2384,25 +2393,41 @@ def check_element_tooltip(self, element, expected_text, contains=False):
         >>> # Call the method:
         >>> has_add_text = self.check_element_tooltip(button_object, "Add")
         """
-
+        
         has_text = False
+        expected_text = re.sub(' ', '', expected_text.lower())
+        element_function = False
+
+        if self.webapp_shadowroot():
+            tooltip_term = 'wa-tooltip, .po-tooltip'
+            if type(element) == Tag:
+                element = self.soup_to_selenium(element)
+            if self.execute_js_selector('button', element):
+                element_function = lambda: next(iter(self.execute_js_selector('button', element)))
+            elif element:
```

Comment:
a função execute js selector possui um parametro para capturar apenas o proximo item, sem que seja necessario utilizar o next(iter
```suggestion
            if self.execute_js_selector('button', element):
                element_function = lambda: self.execute_js_selector('button', element , get_all = False)
            elif element:
```